src/Features/CSharp/Portable/ConvertSwitchStatementToExpression/ConvertSwitchStatementToExpressionCodeFixProvider.Rewriter.cs

```diff
@@ -253,15 +202,15 @@ private ExpressionSyntax RewriteSwitchStatement(SwitchStatementSyntax node, bool
                     Token(leading: default, SyntaxKind.SwitchKeyword, node.CloseParenToken.TrailingTrivia),
```

Comment:
>`node.Expression.Parenthesize()`

This was added in https://github.com/dotnet/roslyn/pull/37016. I think we should only do this if it's an expression with lower precedence than a switch expression.